apps/backend/core/configuration/validation/utils/database.py

```
@@ -0,0 +1,362 @@
+                return False, "Query execution validation failed", 0.0, metadata
+
+            # Get database name
+            db_name = await conn.fetchval("SELECT current_database()")
+            metadata["database"] = db_name
+
+            # Get connection info
+            metadata["pool_size"] = pool.get_size()
+            metadata["pool_free"] = pool.get_idle_size()
+
+        latency_ms = (time.time() - start_time) * 1000
+        return True, None, latency_ms, metadata
```

Ruff (0.14.4)

Consider moving this statement to an `else` block

(TRY300)